### ISSUE TITLE
Optimize images

### ISSUE THREAD

**Issue by swapagarwal** [swapagarwal/swag-for-dev#57]
Images take too long to load on slower networks!
They can be compressed and stored on the server itself, instead of loading it every time from different servers.

**Comment by swapagarwal** [comment]
TL;DR Add images under assets and update the links in `data.json`

**Comment by plibither8** [comment]
Hotlinking can be an issue too (though ideally shouldn't be). The source of the image should also be captioned, IMHO.

**Comment by swapagarwal** [comment]
Nope. We don't have such high traffic right now for hotlinking to be an issue. It's more about images being large (300KB for example)! Source of the image would almost always be present in the reference link so we can omit that.

**Comment by plibither8** [comment]
Ah alright!

**Comment by aslafy-z** [comment]
I'd be pleased to fix this if you haven't already did it!

We could download these images from the gulp pipeline and then rebuild a modified version of data.json to be used by the website. It would allow us to keep original reference to the image in the data.json inside the repo and only edit this json file.

In the future, the README.md could also be generated from this unique json file.

What do you think?

**Comment by swapagarwal** [comment]
Sure, go ahead @aslafy-z!

**Comment by aslafy-z** [comment]
Great! I'll have to embed the `data.json` file into the netlify build to make it work. It will also speed-up the thing.
